app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      The form contains <%= pluralize(@user.errors.count, "error") %>.
```

Comment:
Sử dụng I18n và cách dùng param trong I18n em nhé

2. Phân biệt count, length, size nhé
Cân nhắc tại đây dùng count là hợp lý nhất chauw

Reply:
- count: trả về số lượng phần tử bằng cách đếm các bản ghi từ database (SELECT COUNT (*)  FROM), bên cạnh đó có thể đếm số lượng phần tử kèm theo điều kiện.
- length: trong lần đếm đầu tiên sẽ phải thực hiện load toàn bộ databse để đếm (cần gọi all), kể từ lần đếm thứ 2 thì sẽ nhanh hơn khi chỉ cần đếm trong mảng hiện tại đã được lưu ở bộ nhớ.
- size: kết hợp giữa count và length, ở lần đếm đầu tiên sẽ được thực hiện giống count, kể từ lần thứ 2 thì thực hiện giống length.
=> Dùng size là tối ưu